### PR TITLE
fix: bump go.mod path to v2

### PR DESCRIPTION
**Pull request by shanduur**
The path in `go.mod` should be ended in `/v2` suffix, as per [go.mod module version numbers](https://go.dev/doc/modules/version-numbers).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <img src="https://img.shields.io/badge/licence-APL2-brightgreen"
          alt="license">
   </a>
-  <a href="https://github.com/kairos-io/provider-kairos/issues"><img src="https://img.shields.io/github/issues/kairos-io/provider-kairos"></a>
+  <a href="https://github.com/kairos-io/provider-kairos/v2/issues"><img src="https://img.shields.io/github/issues/kairos-io/provider-kairos"></a>
```
Comment (mauromorales) [inline README.md:14]: This one does not need to have the v2 in it

**Review by mauromorales — CHANGES_REQUESTED** [review]
Only the change of the issues URL needs to be reverted, other than that LGTM 💪

File: README.md (hunk 1 above)
Comment (shanduur) [reply README.md:14]: Ah, whoops, my bad!

**Review by mauromorales — APPROVED** [review]
lgtm

**Pull request merged**